src/core/Notification/BaseNotification/BaseNotification.tsx

```diff
@@ -0,0 +1,34 @@
+import { ReactNode } from 'react';
+import { HtmlDivWithRefProps } from '../../../reset';
+
+export const baseClassName = 'fi-notification';
+
+export const notificationClassNames = {
+  styleWrapper: `${baseClassName}_style-wrapper`,
+  content: `${baseClassName}_content`,
+  label: `${baseClassName}_label`,
+  textContentWrapper: `${baseClassName}_text-content-wrapper`,
+  icon: `${baseClassName}_icon`,
+  iconWrapper: `${baseClassName}_icon-wrapper`,
+  closeButton: `${baseClassName}_close-button`,
+  closeButtonWrapper: `${baseClassName}_close-button-wrapper`,
+  smallScreen: `${baseClassName}--small-screen`,
+  inline: `${baseClassName}--inline`,
+  actionElementsWrapper: `${baseClassName}_actions-elements-wrapper`,
```

Comment:
I would drop the plural from the name since it can be one or more elements. So just `actionElementWrapper` to be grammatically correct.

Reply:
Plurals removed.